.github/workflows/ci.yml

```diff
@@ -481,25 +421,11 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Authenticate to Google Cloud
-        uses: google-github-actions/auth@v2
-        with:
-          credentials_json: ${{ secrets.GCP_SA_KEY }}
-
-      - name: Download native/bin artifact
-        uses: ./.github/actions/gcs-download-cloud-storage
-        with:
-          path-prefix: ${{ github.run_id }}/${{ matrix.name }}
-          source: depends/${{ matrix.host }}/native/bin
-          destination: ./depends/${{ matrix.host }}/native/
```

Comment:
Is this not necessary for the `test-secp256k1` job anymore? Or was it never required (and only `test-univalue` needed it)?

Reply:
It seems it wasn't necessary, and we were downloading these artifacts unnecessarily. At least in the last tests, it worked fine without any issues